numerical_methods/false_position.cpp

```diff
@@ -4,71 +4,93 @@
  * method](https://en.wikipedia.org/wiki/Regula_falsi), also known as the Secant
  * method
  *
- * Given two points \f$a\f$ and \f$b\f$ such that \f$f(a)<0\f$ and
+ * First, multiple intervals are selected with the interval gap provided. 
+ * Separate recursive function called for every root.
+ * Roots are printed Separatelt.
+ *
+ * For an interval [a,b] \f$a\f$ and \f$b\f$ such that \f$f(a)<0\f$ and
  * \f$f(b)>0\f$, then the \f$(i+1)^\text{th}\f$ approximation is given by: \f[
  * x_{i+1} = \frac{a_i\cdot f(b_i) - b_i\cdot f(a_i)}{f(b_i) - f(a_i)}
  * \f]
  * For the next iteration, the interval is selected
  * as: \f$[a,x]\f$ if \f$x>0\f$ or \f$[x,b]\f$ if \f$x<0\f$. The Process is
  * continued till a close enough approximation is achieved.
  *
- * \see newton_raphson_method.cpp, bisection_method.cpp
```

Comment:
Please don't remove this (re-add it again); this is useful when generating the documentation.